src/workerd/io/actor-storage.capnp

```diff
@@ -51,7 +54,7 @@ interface ActorStorage @0xd7759d7fc87c08e4 {
     txn @0 (settings :DbSettings) -> (transaction :Transaction);
 
     interface Transaction extends(Operations) {
-      commit @0 ();
+      commit @0 (ExpiryMsSinceUnixEpoch :UInt64);
       rollback @1 ();
```

Comment:
Why doesn't `rollback` need a expiration timestamp?  Is the idea that something that's rolled back hasn't actually hit storage yet so we don't care about the session?

Reply:
Yes, I don't think rollback will be able to affect anything. It won't be able to affect a new session, since that new session shouldn't have a way of knowing about previous uncommitted transactions.